maventa/types/invoice.ts

```diff
@@ -0,0 +1,46 @@
+interface Invoice {
+    id: string;
+    status: string;
+    reference: string | null;
+    number: string;
+    sender: {
```

Comment:
You'll find it much easier to define own type for each object inside the interface. It's extra work up front, but will save time a lot later.